js/rpg_managers/ImageManager.js

```diff
@@ -9,6 +9,14 @@ function ImageManager() {
 
 ImageManager.cache = new CacheMap(ImageManager);
```

Comment:
I wanna remove it, but We'd better mark obsolete for a short time, then remove it.